components/dropdown/test/.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+	"extends": "brightspace/wct-lit-config"
+}
```

Comment:
Sorry as I know this stuff wasn't ready when you opened the PR -- and feel free to do it in a follow-up, but could you convert these to the new open-wc based tests? Use the button tests as an example.